projects/schematics/src/migrations/5_0/constructor-deprecations/data/import-entries-dialog.component.migration.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  ELEMENT_REF,
+  IMPORT_ENTRIES_DIALOG_COMPONENT,
+} from '../../../../shared/constants';
+import {
+  SPARTACUS_CART_IMPORT_EXPORT_COMPONENTS,
+  SPARTACUS_CORE,
+} from '../../../../shared/libs-constants';
+import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
+
+export const IMPORT_ENTRIES_DIALOG_COMPONENT_MIGRATION: ConstructorDeprecation =
+  {
+    //feature-libs/cart/import-export/components/import-to-cart/import-entries-dialog/import-entries-dialog.component.ts
+    class: IMPORT_ENTRIES_DIALOG_COMPONENT,
+    importPath: SPARTACUS_CART_IMPORT_EXPORT_COMPONENTS,
+    deprecatedParams: [
+      {
+        className: IMPORT_ENTRIES_DIALOG_COMPONENT,
+        importPath: SPARTACUS_CART_IMPORT_EXPORT_COMPONENTS,
```

Comment:
I think you have a mistake here. I suggested removing LAUNCH_DIALOG_SERVICE from `addParams`, it should stay here. I recommend github's "commit suggestion" option for avoiding confusion (then manually fix imports etc)
```suggestion
        className: LAUNCH_DIALOG_SERVICE,
        importPath: SPARTACUS_STOREFRONTLIB,
```

Reply:
done